primitives/account/src/account/staking_contract/validator.rs

```diff
@@ -464,9 +481,9 @@ impl StakingContract {
         // Check that the validator is not jailed.
         if validator.is_jailed(block_number) {
             debug!(
-                "Validator {} is jailed until {}",
+                "Validator {} is jailed since {}",
```

Comment:
Use `tracing` style:
```suggestion
                ?validator_address,
                jailed_from = validator.jailed_from.unwrap(),
                "Validator is jailed",
```